packages/paste-website/src/pages/patterns/pricing/index.mdx

```diff
@@ -0,0 +1,390 @@
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>
+        <strong>Heading</strong>
+      </Td>
+      <Td>
+        This is a smaller heading used for displaying the plan name or other details if including the plan name as the display heading.
+        Refer to the <Anchor href="/components/heading#accessibility">heading documentation</Anchor> for accessibility of headers.
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>
+        <strong>Display Heading</strong>
+      </Td>
+      <Td>
+        Use this largest heading for displaying the price of the plan. If the price is not available, it can be used to display the plan name.
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>
+        <strong>Paragraph subheader</strong>
+      </Td>
+      <Td>
+        The subheader can be placed above or below the display heading. Use this to add additional context to the price or plan.
+        For example “Included” or “Plus fees” or “Best for XYZ”
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>
+        <strong><Anchor href="/components/button">Button</Anchor> (card variant)</strong>
+      </Td>
+      <Td>
+        Use a secondary button to select the plan. We suggest using the button text “Select plan”.
+        For the current plan, hide the button or show an CTA. For example, it could navigate to the feature the user has already selected.
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>
+        <strong>Feature list (<Anchor href="/components/icon/usage-guidelines">icon</Anchor> + <Anchor href="/components/paragraph">paragraph</Anchor>)</strong>
+      </Td>
+      <Td>
+        Use this list to outline the differences between each plan. Include 1-8 features, the first feature listed should be anything included in previous features (if any).
+        For example “Packaged plan 1 features”
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>
+        <strong>More info button (card variant)</strong>
+      </Td>
+      <Td>
+        Use an optional link at the bottom of the card to link to additional documentation or details about the plan that can’t fit in the card.
+        This should go to a new page or open a side modal, not expand the card. Follow <Anchor href="/components/button">Paste guidelines</Anchor> for using a button or anchor: Use a link style button when the user is performing an action to open a modal or side modal, and an anchor when it’s navigating to a new page
+      </Td>
+    </Tr>
+  </TBody>
+</Table>
+
+## Responsiveness
+
+For larger viewports, cards and pickers should be side-by-side. For smaller viewports, stack the cards or picker vertically.
+
+## Figma
+
+The **pricing card group** Figma component has min and max widths already applied.
+
+However the height of the cards should all be equal to each other even if the number of features, badges, or buttons are different.
+You may need to adjust the height of the card to “fill” in Figma. **The “More Info” button should remain at the bottom of the card.**
```

Comment:
```suggestion
You may need to detach the component, set autolayout to "Horizontal layout" (not "Wrap"), and adjust the height of the card to “Fill” in Figma. **The “More info” button should remain at the bottom of the card.**
```

Reply:
nice, yeah I like that. And it's an easier solution than asking people to detach and fiddle around with autolayout.

The only change we still need is recommending the actual values for min/max widths ([Figma comment thread here](https://www.figma.com/design/CdAGlhnT0JdVyExeEGZ12y?node-id=10385-2184#968709247)). @PixeledCode what was the min/max width issue you were running into?